src/harberger-lib/components/BuyInput.res

```diff
@@ -1,15 +1,14 @@
 type debounceOptions = {
-  @dead("debounceOptions.maxWait") maxWait: option<int>,
-  @dead("debounceOptions.leading") leading: option<bool>,
-  @dead("debounceOptions.trailing") trailing: option<bool>,
+  @live maxWait: option<int>,
+  @live leading: option<bool>,
+  @live trailing: option<bool>,
 }
 
+type callBack<'callback> = {callback: 'callback, cancel: unit => unit, flush: unit => unit}
+
 @module("use-debounce")
-external useDebouncedCallback: (
-  . 'callback,
-  int,
-  debounceOptions,
-) => ('callback, unit => unit, unit => unit) = "useDebouncedCallback"
+external useDebouncedCallback: (. 'callback, int, debounceOptions) => callBack<'callback> =
+  "useDebouncedCallback"
```

Comment:
The api changed :/

I updated the npm package without checking. Anyway now using the latest api, all is good :+1: 
![image](https://user-images.githubusercontent.com/6032276/104457008-23514380-55b2-11eb-8c78-c8fcd4dc3830.png)